quantecon/markov/approximation.py

```diff
@@ -17,46 +17,71 @@
 
 
 def rouwenhorst(n, ybar, sigma, rho):
-    """
-    Takes as inputs n, p, q, psi.  It will then construct a markov chain
+    r"""
+    Takes as inputs n, p, q, psi. It will then construct a markov chain
     that estimates an AR(1) process of:
-    y_t = \bar{y} + \rho y_{t-1} + \varepsilon_t
-    where \varepsilon_t is i.i.d. normal of mean 0, std dev of sigma
+    :math:`y_t = \bar{y} + \rho y_{t-1} + \varepsilon_t`
+    where :math:`\varepsilon_t` is i.i.d. normal of mean 0, std dev of sigma
```

Comment:
Apparently they look slightly different but represent the same symbol - https://tex.stackexchange.com/questions/304534/what-does-the-var-prefix-stand-for-in-varphi-and-varepsilon-etc